workflow-testing/src/main/java/com/squareup/workflow/testing/RenderTester.kt

```diff
@@ -220,9 +223,70 @@ interface RenderTester<PropsT, StateT, OutputT, RenderingT> {
   /**
    * Specifies that this render pass is expected to render a particular child workflow.
    *
+   * ## Expecting impostor workflows
+   *
+   * Identifiers are compared using [WorkflowIdentifier.matchesActualIdentifierForTest]. If the
+   * workflow-under-test renders an [ImpostorWorkflow][com.squareup.workflow.ImpostorWorkflow],
+   * the expected [identifier] must match the leaf real identifier of the `ImpostorWorkflow`.
+   *
+   * For example, given the `mapRendering` workflow operator that returns an `ImpostorWorkflow`
```

Comment:
Documentation nit: hopefully most folks won't need to know the impostor workflow details, so maybe you want to put the example above this section. Then this heading can be something more like `## Implementation details: imposter workflows`.

If you want to just get this merged, I'm happy to take an editorial pass later.